types/react-native/index.d.ts

```diff
@@ -4362,9 +4363,22 @@ export interface SectionListStatic<SectionT> extends React.ComponentClass<Sectio
     scrollToLocation?(params: SectionListScrollParams): void;
 }
 
+/**
+ * @see https://facebook.github.io/react-native/docs/virtualizedlist.html
+ */
+
+export class VirtualizedList<ItemT> extends React.Component<VirtualizedListProps<ItemT>> {
+    scrollToEnd: (params?: { animated?: boolean }) => void;
+    scrollToIndex: (params: { animated?: boolean; index: number; viewOffset?: number; viewPosition?: number }) => void;
+    scrollToItem: (params: { animated?: boolean; item: ItemT; viewPosition?: number }) => void;
+    scrollToOffset: (params: { animated?: boolean; offset: number }) => void;
```

Comment:
Can you copy the documentation of this method here as JSDoc?

Reply:
Added
- [scrollToOffset](https://reactnative.dev/docs/virtualizedlist#scrolltooffset)

This item doesn't have any explaination.
- [scrollToEnd](https://reactnative.dev/docs/virtualizedlist#scrolltoend)

These items explaination doesn't seem to be useful.
- [scrollToIndex](https://reactnative.dev/docs/virtualizedlist#scrolltoindex)
- [scrollToItem](https://reactnative.dev/docs/virtualizedlist#scrolltoitem)

Reply:
Perfect; I only meant the one you added 👍